FF1Lib/Flags.cs

```diff
@@ -898,8 +1021,30 @@ public static Flags DecodeFlagsText(string text)
 		{
 			var sum = StringToBigInteger(text);
 
+			var tempExtensiveHints_BinMatrix = new bool[Enum.GetValues(typeof(HintCategory)).Length][];
+
+			for (int i = 0; i < tempExtensiveHints_BinMatrix.Length; i++) tempExtensiveHints_BinMatrix[i] = new bool[Enum.GetValues(typeof(HintLocation)).Length];
+
+			//You've got to be fucking shitting me. I have to do it in reverse?

```

Comment:
Obviously the flags do need to be decoded in reverse of the order they are encoded since the whole thing is just one bigass number written down as a hex string. I think you can remove the comment.

Reply:
Generally humorous comments and variable names are tolerated but this is probably a bridge too far.

Reply:
Sorry.

Reply:
removed

Reply:
When serializing and deserializing things, you usually do it in the same order. While i understand the implementation and the reason for it, it is very unintuitive. Contributors should not have to learn the quirks of a specific implementation. Therefore i come tho think, that there actually should be a comment. But that is not the scope of this PR.